website/docs/auth/service_principal_client_secret.html.markdown

```diff
@@ -1,17 +1,24 @@
 ---
 layout: "azurerm"
-page_title: "Azure Provider: Authenticating via a Service Principal"
-sidebar_current: "docs-azurerm-index-authentication-service-principal"
+page_title: "Azure Provider: Authenticating via a Service Principal and a Client Secret"
+sidebar_current: "docs-azurerm-authentication-service-principal-client-secret"
 description: |-
-  This guide will cover how to use a Service Principal (Shared Account) as authentication for the Azure Provider.
+  This guide will cover how to use a Service Principal (Shared Account) with a Client Secret as authentication for the Azure Provider.
 
 ---
 
-# Azure Provider: Authenticating using a Service Principal
+# Azure Provider: Authenticating using a Service Principal with a Client Secret
 
-Terraform supports authenticating to Azure through a Service Principal or the Azure CLI.
+Terraform supports a number of different methods for authenticating to Azure:
 
-We recommend using a Service Principal when running in a shared environment (such as within a CI server/automation) - and [authenticating via the Azure CLI](authenticating_via_azure_cli.html) when you're running Terraform locally.
+* [Authenticating to Azure using the Azure CLI](azure_cli.html)
+* [Authenticating to Azure using Managed Service Identity](managed_service_identity.html)
+* [Authenticating to Azure using a Service Principal and a Client Certificate](service_principal_client_certificate.html)
+* Authenticating to Azure using a Service Principal and a Client Secret (which is covered in this guide)
+
+---
+
+We recommend using either a Service Principal or Managed Service Identity when running in automation (for example on a CI Server) - and authenticating using the Azure CLI when running Terraform locally.
```

Comment:
This wording confuses me "...running in automation...", maybe "...running Terraform via automation..." or something along those lines?

Reply:
perhaps instead of automation we could use the term non-interactive?